tests/standalone/bug0655.phpt

```diff
@@ -0,0 +1,46 @@
+--TEST--
+PHPC-655: Use case insensitive parsing for Manager connectTimeoutMS array option
+--SKIPIF--
+<?php if (defined("HHVM_VERSION_ID")) exit("skip HHVM does not use custom streams"); ?>
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$command = new MongoDB\Driver\Command(['ping' => 1]);
+
+// Invalid host cannot be resolved
+$manager = new MongoDB\Driver\Manager('mongodb://invalid.host:27017', ['connectTimeoutMS' => 1]);
```

Comment:
I didn't see the point, as the code (our own and `mongoc-uri.c`) uses all-lowercase string literals. I can add a few, though.